docs/source/index.rst

```diff
@@ -25,6 +25,7 @@ User Guide
 
 .. toctree::
 
+   guide/upgrading
```

Comment:
Should this just be added to to `guide/index.rst`?  There's already sections on migrating.  Seems like we can just build on that.